sources/packages/forms/src/form-definitions/sfaa2023-24.json

```diff
@@ -32485,8 +32642,173 @@
\n\t<li>I will immediately notify the financial aid office of any changes to the information initially submitted by myself or my spouse/common-law partner. This includes my or my partners financial information (including income), changes to address, academic status, marital status, number of dependents, or any other information captured on the application form.&nbsp;</li>\n\t<li>I consent to the exchange of information between StudentAid BC, the post-secondary institution and/or other appropriate financial aid office about my academic standing, awards, program of study, course load, attendance, living arrangements, marital status and financial status. This consent takes effect when I sign this declaration.</li>\n\t<li>I consent to the verification of my Social Insurance Number, name, date of birth and gender, with information contained in Employment and Social Development Canada Social Insurance Registry. This information will be disclosed to Employment and Social Development Canada for the purpose of confirming the accuracy of my identification in the context of this application for a Canada Student Loans and Grants for part-time studies. This consent takes effect when I sign this declaration.</li>\n\t<li>For the purpose of verifying and/or investigating information pertaining to this application, related documents, and the eventual repayment of my loan awards, whether defaulted or not and any other money repayable, I consent to the collection, use and disclosure of my personal information between the BC Ministry of Post-Secondary Education and Future Skills, the BC Ministry of Finance, National Student Loans Service Centre, and any of their contractors, subcontractors or agents, each with each other, and with the following: financial institutions, educational institutions, &nbsp;employers, credit bureaus, credit reporting agencies, indigenous organizations, federal and provincial Crown corporations, and federal, provincial, municipal ministries/departments/agencies, including but not limited to: BC Ministry of Social Development and Poverty Reduction, BC Ministry of Children and Family Development, BC Ministry of Health, BC Ministry of Attorney General, BC Ministry of Education and Child Care, BC Public Service Agency, RoadSafety BC, Insurance Corporation of BC (and Service BC acting in the role of ICBC), BC Hydro, BC Assessment Authority, Land Title and Survey Authority of BC, BC Registry Services, WorkSafeBC, BC Vital Statistics Agency, the Office of the Superintendent of Bankruptcy Canada, Employment and Social Development Canada, Canada Revenue Agency, Immigration, Refugees and Citizenship Canada. This consent takes effect on the date that I submit this Application to StudentAid BC.</li>\n\t<li>I have not applied nor will I apply to or receive government funded student loans, grants or bursaries from another province or territory or government funded loans outside of Canada during the study period stated in this application.</li>\n</ol>\n",
```

Comment:
Please justify the text as the Word document:
![image](https://github.com/bcgov/SIMS/assets/78114138/ae6f1ebb-0eee-48f8-a1e2-d899bbe2e421)

Reply:
Thanks for spotting it. I'm verifying with Hugh and will correct it accordingly.

Reply:
@andrepestana-aot Hugh's response is that "left-align" is totally fine at this point. Formatting decisions like this are at our discretion ... unless there are impacts to the wording itself. I think that "left-align" should be okay for this paragraph, so we don't have to use "justify" for now. Let me know if this solves your concern.

Reply:
That's ok then. Thanks for checking.